bi-connectors/PowerBIConnector/OpenSearchProject-ODBC.pq

```diff
@@ -205,14 +205,14 @@ OpenSearch = [
 
     // PBIDS Handler
     DSRHandlers = [
-      #"opensearch-sql" = [
-        GetDSR = (server, schema, object, optional options) => [ protocol = "opensearch-sql", address = [ server = server, port = schema, useSSL = object ] ],
+      #"opensearchproject-odbc" = [
+        GetDSR = (server, schema, object, optional options) => [ protocol = "opensearchproject-odbc", address = [ server = server, port = schema, useSSL = object ] ],
         GetFormula = (dsr, optional options) => () =>
           let
-            db = OpenSearch.Contents(dsr[address][server], dsr[address][port], dsr[address][useSSL])
+            db = #"OpenSearchProject-ODBC.Contents"(dsr[address][server], dsr[address][port], dsr[address][useSSL])
           in
             db,
-        GetFriendlyName = (dsr) => "OpenSearch SQL ODBC"
+        GetFriendlyName = (dsr) => "OpenSearch Project SQL ODBC"
```

Comment:
Why do we have SQL? Suggest just OpenSearch Project

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/27/commits/f27402a988e667b46f62712520cd5337128a3008